src/main/java/org/example/MethodsForStudents.java

```diff
@@ -0,0 +1,81 @@
+package org.example;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class MethodsForStudents {
+    JDBS jdbs = new JDBS();
```

Comment:
Shouldn't be in class level

Reply:
What exactly?

Reply:
Declaration of this variable

Reply:
Can I declare this variable in each method?

Reply:
Sure